tests/migrations/test_operations.py

```diff
@@ -5929,6 +5929,41 @@ def test_add_field_after_generated_field(self):
         self.assertEqual(pony_new.generated, 1)
         self.assertEqual(pony_new.static, 2)
 
+    def test_change_id_pk_to_uuid_pk(self):
```

Comment:
```suggestion
    def test_alter_id_pk_to_uuid_pk(self):
```
I'd move closer to other tests related with primary key alterations.